doc/release-notes-1.1.0.rst

```diff
@@ -0,0 +1,621 @@
+.. _ncs_release_notes_110:
+
+|NCS| v1.1.0 Release Notes
+##########################
+
+|NCS| delivers reference software and supporting libraries for developing low-power wireless applications with Nordic Semiconductor products.
+It includes the MCUboot and the Zephyr RTOS open source projects, which are continuously integrated and re-distributed with the SDK.
+
+|NCS| v1.1.0 supports product development with the nRF9160 Cellular IoT device.
+It contains reference applications, sample source code, and libraries for Bluetooth Low Energy devices in the nRF52 Series, though product development on these devices is not currently supported with the |NCS|.
+
+.. important::
+   |NCS| v1.1.0 does not provide support for nRF5340.
+   For nRF5340 support, check out the master branch.
+
+Highlights
+**********
+
+
+
+Release tag
+***********
+
+The release tag for the |NCS| manifest repository (|ncs_repo|) is **v1.1.0**.
+Check the ``west.yml`` file for the corresponding tags in the project repositories.
+
+To use this release, check out the tag in the manifest repository and run ``west update``.
+See :ref:`Getting the nRF Connect SDK code <cloning_the_repositories_win>` for more information.
+
+
+Supported modem firmware
+************************
+
+* mfw_nrf9160_1.1.0
+
+Use the nRF Programmer app of `nRF Connect for Desktop`_ to update the modem firmware.
+See `Updating the nRF9160 DK cellular modem`_ for instructions.
+
+
+Supported boards
+****************
+
+* PCA10090 (nRF9160 DK)
+* PCA20035 (Thingy:91)
+* PCA10056 (nRF52840 Development Kit)
+* PCA10059 (nRF52840 Dongle)
+* PCA10040 (nRF52 Development Kit)
```

Comment:
What about 52833?

Reply:
No I dont think we should mention this. Release branch doesnot have nrf52833_pca10100 support.